src/server/tran_server.cpp

```diff
@@ -254,19 +253,16 @@ tran_server::init_page_server_hosts ()
   //
   int valid_connection_count = 0;
   bool failed_conn = false;
-  for (const cubcomm::node &node : m_connection_list)
+  for (const auto &conn : m_page_server_conn_vec)
     {
-      /* create a empty connection_handler specialized for each tran serve type */
-      m_page_server_conn_vec.emplace_back (create_connection_handler (*this));
-      exit_code = m_page_server_conn_vec.back ()->connect (node);
+      exit_code = conn->connect ();
       if (exit_code == NO_ERROR)
 	{
 	  ++valid_connection_count;
 	}
       else
 	{
 	  failed_conn = true;
-	  er_log_debug (ARG_FILE_LINE, "Failed to connect to host: %s port: %d\n", node.get_host ().c_str (), node.get_port ());
```

Comment:
tran_server doesn't know the node, and a similar log is written in `tran_server::connection_handler::connect`.